examples/advanced_circuits_algorithms/Deutsch_Jozsa_Algorithm/Deutsch Jozsa Algorithm.ipynb

```diff
@@ -0,0 +1,1047 @@
+{
```

Comment:
<p>Maybe change 'For the two possible cases, if the function is constant then the measurement will observe all 0’s, if the function is balanced then you will observe a mix of 1's and 0's and if you have both measurements present for a function the it's neither constant or balanced.' to 'For the two possible cases, if the function is constant then the measurement will observe all 0’s for the first qubit register and if the function is balanced then you will observe an equal mix of 1's and 0's for the first register and if you have both types of measurements present for a function the it's neither constant or balanced.'</p><p> </p><p>Also, this is a standard mistake that everybody makes but you need to get your mathematical latex to work properly. So, for instance, $Uf$ should be $U_f$. Latex is not fun at first!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/177/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Updated in most recent PR.